gen/build_deploy/powershell/dcos_generate_config_win.sh.in

```diff
@@ -8,6 +8,18 @@
 #
 set -o errexit -o nounset -o pipefail
 
+# check if dcos-config-win package exists:
+if [ -f ./genconf/serve/cluster-package-info.json ]; then
+    exists="$(grep -Po '"dcos-config-win":*' ./genconf/serve/cluster-package-info.json)";
+    if ! [ "$exists" ] ; then
+       >&2 echo "ERROR: 'dcos-config-win' package is missing! Possible cause - 'enable_windows_agents: true' not specified while DC/OS installer setup. Please re-run setup with such flag!";
+       exit 1;
+    fi
+else
+    >&2 echo "ERROR: No ./genconf/serve/cluster-package-info.json exists to validate DC/OS installer setup";
```

Comment:
This suggests that the Linux installer has not been run first. Maybe we should add a suggestion to do that?

Reply:
`ERROR: 'dcos-config-win' package is missing! Possible cause - 'enable_windows_agents: true' not specified while DC/OS Linux installer setup. Please set the flag first and run DC/OS Linux Installer setup again!`

@jongiddy 
Does it sound full enough?